scss/_footer.scss

```diff
@@ -1,167 +1,164 @@
-
-      > * {
-        padding-right: 0;
-        padding-left: 0;
-      }
-    }
+  --#{$prefix}footer-part-gap: #{$footer-gap-xl};
+  --#{$prefix}navbar-nav-padding-top: #{$footer-nav-list-padding-top};
+  --#{$prefix}navbar-nav-padding-x: 0;
+  --#{$prefix}navbar-nav-padding-bottom: #{$footer-nav-list-padding-bottom};
+
+  .accordion {
+    --#{$prefix}accordion-btn-font-size: #{$footer-font-size-sm};
+    --#{$prefix}accordion-btn-line-height: #{$footer-accordion-line-height};
+    --#{$prefix}accordion-btn-letter-spacing: #{$footer-letter-spacing};
   }
 
   .footer-column:first-of-type .accordion-header:first-of-type,
   .accordion .accordion-item {
-    border: 0;
-  }
-
-  .accordion-header {
-    @include media-breakpoint-up(md) {
-      padding: $footer-nav-padding-top 0 $spacer;
-      border: 0;
-    }
-
-    .accordion-button {
-      font-size: $footer-font-size-sm;
-      line-height: $footer-accordion-line-height;
-      letter-spacing: $footer-letter-spacing;
-
-      &::after {
-        margin-right: 0;
-      }
-
-      @include media-breakpoint-up(md) {
-        all: unset;
-        pointer-events: none;
-
-        &::after {
-          content: unset;
-        }
-      }
-    }
-  }
-
-  .navbar-nav {
-    gap: $footer-gap-xl;
-    padding: $footer-nav-list-padding-top 0 $footer-nav-list-padding-bottom;
-
-    @include media-breakpoint-up(md) {
-      flex-direction: column;
-      gap: $footer-gap;
-      padding-top: $footer-nav-list-padding-top-md;
-      padding-bottom: $footer-nav-list-padding-bottom-md;
-      font-size: $footer-font-size-md;
-      line-height: $footer-line-height-md;
-    }
-
-    @include media-breakpoint-up(lg) {
-      padding-bottom: $footer-nav-list-padding-bottom-lg;
-    }
+    --#{$prefix}accordion-border-width: 0;
   }
 
-  .accordion-collapse {
-    @include media-breakpoint-up(md) {
-      display: flex;
-      padding-left: 0;
-    }
+  .accordion-button::after {
+    margin-right: 0;
   }
 }
 
 .footer-service {
-  padding-top: subtract($footer-service-padding-y, .0625rem);
-  padding-bottom: $footer-service-padding-y;
-
-  @include media-breakpoint-up(md) {
-    padding-top: add($footer-service-padding-y-md, .0625rem);
-    padding-bottom: $footer-service-padding-y-md;
-  }
+  --#{$prefix}footer-part-padding-top: #{subtract($footer-service-padding-y, .0625rem)};
+  --#{$prefix}footer-part-padding-bottom: #{$footer-service-padding-y};
 
   .nav-link > span {
     padding-top: $footer-service-link-padding-top;
   }
 }
 
 .footer-terms {
-  padding-top: $footer-terms-padding-top;
-  padding-bottom: $footer-terms-padding-bottom;
+  --#{$prefix}footer-part-padding-top: #{$footer-terms-padding-top};
+  --#{$prefix}footer-part-padding-bottom: #{$footer-terms-padding-bottom};
+  --#{$prefix}footer-part-gap: #{$footer-gap-xl};
+}
 
-  @include media-breakpoint-up(md) {
-    padding-top: $footer-terms-padding-y-md;
-    padding-bottom: $footer-terms-padding-y-md;
-  }
+.navbar-expand {
```

Comment:
Took this class from `_navbar.scss`. Not sure that it should be the case since there could be some interference, however, it helps save some bundle size. 
I can either : 
- rename it `.footer-expand` and backport all what is needed in this class
- remove this behavior for the footer
- keep it as shown here

Reply:
If you only load `_footer.scss` and not `_navbar.scss` in a project is it still working?
If so you could add something like `.footer.navbar-expand` to specify it more.